src/components/SquaresGrid.js

```diff
@@ -1,9 +1,34 @@
 import React from 'react'
+import styled from 'styled-components'
+
+const zero_to_nine = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+export const SquaresRow = styled.div`
+  display: flex;
+`
+
+export const SquaresColumn = styled.div`
+  flex: 1;
+`
 
 export const SquaresGrid = () => {
   return (
-    <div>
-      <h1>Square Grid</h1>
+    <div className="squares">
```

Comment:
How is this to read?

I've done what I could to use classNames and better property names but it still seems kind of messy.

I could also take out the word `Squares` from each of these. What do you think @mspremulli ?